src/openrct2/scripting/bindings/entity/ScEntity.hpp

```diff
@@ -41,7 +41,7 @@ namespace OpenRCT2::Scripting
         int32_t id_get() const
         {
             auto entity = GetEntity();
-            return entity != nullptr ? entity->sprite_index : 0;
+            return entity != nullptr ? entity->sprite_index.ToUnderlying() : EntityId::GetNull().ToUnderlying();
```

Comment:
This changing the failure to null (which is correct) do we need a plugin version bump?

Reply:
Hm probably, 0 is actually a valid entity id.